README.md

```diff
@@ -31,8 +31,6 @@ The figure below illustrates the software flow through the architecture componen
 
 # Getting Started
 
-Learn how to configure and run the service by following these [instructions](./doc/setup.md). 
```

Comment:
Same comment as other people, what do we want to do with the files left dangling from removing this line?

Reply:
I think for now we can leave these files as it is at least until we migrate everything to edgex docs.